app/views/tube_rack_statuses/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="page-header">
+    <h1>Tube Rack Status</h1>
```

Comment:
Could we call it something like 'Status of tube rack imports' ? That's a bit more meaningful, although does it describe all the use cases?